src/components/Person/Person.jsx

```diff
@@ -1 +1,47 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const ifHavePartnerBlock = (
+    <p className="Person__partner">
+      {partnerName}
+      {' is my '}
+      {sex === 'm'
+        ? 'wife'
+        : 'husband'}
+    </p>
+  );
+
+  const ifNotHavePartnerBlock = (
+    <p className="Person__partner">
```

Comment:
you duplicating `<p className="Person__partner">` tag but you can hold sn variables only content value and place them inside tag in jsx